pkg/storage/internalstorage/resource_storage.go

```diff
@@ -310,3 +317,23 @@ func applyExtraLabelSelectorToResourceQuery_Owner(db *gorm.DB, query *gorm.DB, o
 
 	return query, nil
 }
+
+func applyExtraLabelSelectorToResourceQuery_FuzzySearch(db *gorm.DB, query *gorm.DB, opts *internal.ListOptions) (*gorm.DB, error) {
+	if opts.ExtraLabelSelector == nil {
+		return query, nil
+	}
+
+	if requirements, selectable := opts.ExtraLabelSelector.Requirements(); selectable {
+		for _, require := range requirements {
+			switch require.Key() {
+			case SearchLabelFuzzyName:
```

Comment:
The `require.Operation()` should also be checked, added in the future together with `ListOptions`